### PR TITLE
Locations: make 'name' a required field

### PR DESCRIPTION
**Pull request by cawel**
Fix for issue #109.

### CODE REVIEW

File: db/migrate/20151204170038_change_location_name_to_be_non_nil.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ChangeLocationNameToBeNonNil < ActiveRecord::Migration
+  def up
+    change_column :locations, :name, :string, null: false
```
Comment (nicholasjhenry) [inline db/migrate/20151204170038_change_location_name_to_be_non_nil.rb]: Not super important @cawel but I thought you might like this: `change_column_null`

```ruby
change_column_null(:locations, :name, true)
```

Comment (cawel) [inline db/migrate/20151204170038_change_location_name_to_be_non_nil.rb]: Thanks! I didn't know about that one.

**Comment by nicholasjhenry** [comment]
@cawel This PR requires a rebase please :smile_cat:

**Comment by cawel** [comment]
Oh yeah !  :smile_cat:

**Pull request merged**